packages/payment-detection/test/erc20/fee-proxy-contract.test.ts

```diff
@@ -185,61 +188,88 @@ describe('api/erc20/fee-proxy-contract', () => {
       version: '0.2',
     };
 
-    const mockExtractBalanceAndEvents = () => {
-      return Promise.resolve({
-        balance: '1000',
-        events: [
-          // Wrong fee address
+    const mockExtractTransferEvents = (eventName: any) => {
```

Comment:
IMPORTANT !

these events were wrong imo. It was hidden by the fact the balance was not computed on the events but on the `balance` field.  REVIEW HIGHLY APPRECIATED

Reply:
LGTM indeed, good catch